Server/src/controllers/authenticationController.ts

```diff
@@ -96,15 +94,21 @@ export class AuthenticationController {
   }
 
   private validateUpdatePasswordRequest(request: Request): boolean {
-    return !passwordSchema.isValidSync(request.body.password) && !request.body.passwordConfirmation && request.body.password === request.body.passwordConfirmation;
+    return !request.body.password && !request.body.passwordConfirmation;// && request.body.password === request.body.passwordConfirmation && !passwordSchema.isValidSync(request.body.password);
+  }
+
+  private passwordConfirmationRequest(request: Request): boolean {
+    if (request.body.password === request.body.passwordConfirmation) {
+      return true
+    }
   }
 
   private validateForgotPasswordRequest(request: Request): boolean {
     return !request.body.email;
   }
 
   private validateSignUpRequest(request: Request): boolean {
-    if (emailSchema.isValidSync(request.body.email) && passwordSchema.isValidSync(request.body.password) && request.body.firstName && request.body.lastName && request.body.organizationName) {
+    if (request.body.email && request.body.password && request.body.firstName && request.body.lastName && request.body.organizationName && emailSchema.isValidSync(request.body.email) && passwordSchema.isValidSync(request.body.password)) {
```

Comment:
This is the same as the above. You should split this up

Reply:
split and fixed on latest commit